### PR TITLE
feat. uptime kuma

### PR DESCRIPTION
**Pull request by gOOvER**
# Description

Uptime Kuma is an easy-to-use self-hosted monitoring tool. [https://github.com/louislam/uptime-kuma](https://github.com/louislam/uptime-kuma)

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

## New egg Submissions

1. [x] Does your submission pass tests (server is connectable)?
2. [x] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
3. [x] Have you added a unique README.md for the egg you are adding according to the alphabetical order?

### CODE REVIEW

**Comment by QuintenQVD0** [comment]
there own docker files uses nodejs 16 , please match that: https://github.com/louislam/uptime-kuma/blob/master/docker/debian-base.dockerfile#L3

**Comment by gOOvER** [comment]
> there own docker files uses nodejs 16 , please match that: [louislam/uptime-kuma@`master`/docker/debian-base.dockerfile#L3](https://github.com/louislam/uptime-kuma/blob/master/docker/debian-base.dockerfile?rgh-link-date=2023-01-22T07%3A43%3A59Z#L3)

Their installer use nodejs 14

https://github.com/louislam/uptime-kuma/blob/dd6a76c4cc058a88199d3d601cfc31a6cfcd2790/install.sh#L53

**Comment by gOOvER** [comment]
> there own docker files uses nodejs 16 , please match that: [louislam/uptime-kuma@`master`/docker/debian-base.dockerfile#L3](https://github.com/louislam/uptime-kuma/blob/master/docker/debian-base.dockerfile?rgh-link-date=2023-01-22T07%3A43%3A59Z#L3)

switched to nodejs16

File: software/uptime-kuma/README.md [hunk 1]
```diff
@@ -13,9 +13,16 @@ Egg is based on parkers generic nodejs egg
 Github: https://github.com/louislam/uptime-kuma
 
 ## Admin & login
-
 After installation and successful start, browse to your <ip>:<port> and setup the Admin.
 
+## Integrated Cloudflare Tunnel service
```
Comment (QuintenQVD0) [inline software/uptime-kuma/README.md:18]: this should not be needed in the readme. It is technicly some sort of web hosting

Comment (gOOvER) [reply software/uptime-kuma/README.md:18]: > this should not be needed in the readme. It is technicly some sort of web hosting

Then Gitea is also Webhosting

Comment (QuintenQVD0) [reply software/uptime-kuma/README.md:18]: It is fine as it is running the integrated web server that comes with it. It is the part about contacting you for a difrend image. People come here to get working eggs and not everyone has discord. Also, only official images are supported as if they get your image, and it breaks we get to deal with the issue on the repo here

Comment (gOOvER) [reply software/uptime-kuma/README.md]: fixed with last commit

**Review by QuintenQVD0 — APPROVED** [review]
LGTM! Thx

**Pull request merged**